src/walletSdk/Wallet/index.ts

```diff
@@ -0,0 +1,37 @@
+import FreighterApi from "@stellar/freighter-api";
```

Comment:
Perhaps it's best if we flipped this. This way the frieghter API package could provide the Wallet implementation.

Reply:
Ah yeah, that makes sense. The idea being that the freighter-api is the default interface that we expect from a wallet.

For added context: We're also talking about creating some helper methods in this lib to allow other browser extensions to quickly implement this expected interface in their own codebase so they can quickly make use of `signAndSend` and `signAuthEntries`